src/components/Signin.jsx

```diff
@@ -67,7 +67,7 @@ const AuthLink = styled.a`
   i{
     font-size:${props => props.IconSize};
     color:${props => props.IconColor};
-    float:${props => props.IconFloat};
```

Comment:
Is it working fine now?

Reply:
ya it is working fine now,  wihtout it, and  removed it.